tests/codebuild/run_integtest.sh

```diff
@@ -88,22 +89,27 @@ pushd sagemaker-k8s-operator
     # Setup the PATH for smlogs
     mv smlogs-plugin/linux.amd64/kubectl-smlogs /usr/bin/kubectl-smlogs
 
-    # Goto directory that holds the CRD  
-    pushd sagemaker-k8s-operator-install-scripts
-        # Since OPERATOR_AWS_SECRET_ACCESS_KEY and OPERATOR_AWS_ACCESS_KEY_ID defined in build spec, we will not create new user
-        ./setup_awscreds
-
-        echo "Deploying the operator"
-        kustomize build config/default | kubectl apply -f -
-    popd 
-
+    # Allow for overriding the installation of the CRDs/controller image from the
+    # build scripts if we want to use our own installation
+    if [ "${SKIP_INSTALLATION}" == "true" ]; then
```

Comment:
Nice!

Reply:
Can you write up how to use this in [local run README](https://github.com/aws/amazon-sagemaker-operator-for-k8s/tree/master/tests/codebuild/local-run)?